kuma/dashboards/forms.py

```diff
@@ -13,6 +13,11 @@
     ('week', _('Week')),
     ('month', _('30 days')),
 ]
+AUTHOR_CHOICES = [
+    ('All Authors', _('All Authors')),
```

Comment:
Django will handle the space or the underscore, although I do like avoiding the automatic space conversion to ``All+Authors``.

I would prefer class-level constants, such as:

```python
ALL_AUTHORS=0
KNOWN_AUTHORS=1
UNKNOWN_AUTHORS=2
AUTHOR_CHOICES = [
    (ALL_AUTHORS, _('All Authors')),
    (KNOWN_AUTHORS, _('Known Authors')),
    (UNKNOWN_AUTHORS, _('Unknown Authors')),
]
```
and the usage in the view becomes ``RevisionDashboardForm.ALL_AUTHORS``.